Add section about the security council (#190)
LiskHQ/lisk-docs: docs/docs-user/governance/overview.mdx

import TabItem from '@theme/TabItem';

# Governance overview

> The Lisk project is governed by the [Onchain Foundation](https://onchain.org/foundation/), with support of the [Lisk security council](#lisk-security-council) and the [Lisk DAO](#lisk-dao), which allows Lisk token holders to participate together in the governance of the Lisk project.

Author: ```suggestion
The Lisk project is governed by the [Lisk DAO](#lisk-dao) and [Onchain Foundation](https://onchain.org/foundation/).
The Onchain Foundation defines the vision and strategy for the project together with the Lisk team, funds its development and is a stakeholder in the Lisk DAO via the Lisk Team delegate.
The Lisk DAO allows any LSK token holder that stakes their tokens to participate in onchain governance for the Lisk project.
The Lisk DAO decides how to spend funds in its significant treasury and can pass general proposals about protocol parameters or the project direction.
```

import TabItem from '@theme/TabItem';

# Governance overview

> The security of the Lisk Mainnet is mainly dependent on the [security council](#security-council) and the [Lisk DAO](#lisk-dao), which allows Lisk token holders to participate together in the governance of the Lisk project.

Reviewer: I don't think this is true. The security of Lisk Mainnet is mainly based on ethereum security, OP stack bridge security and sequencer. DAO and security council at this moment does not really contribute to the Network security

Author: True, I updated the statement

Reviewer: I think it's still mixing several things.

Lisk project is managed by Lisk Ltd delegated by Onchain Foundation. (this needs to be checked with Legal team how we should phrase it)
Lisk DAO is managing the DAO funds allocated
Security council is managing the security of the mentioned contracts

Author: DAO is not only managing the funds, it is also possible to propose general proposals that go beyond DAO funds. 
Regarding Lisk Ltd I heard it is similar to Lightcurve and we shouldn't mention it publicly

Author: The admin role of the Onchain foundation is also mentioned on the page below. I am not sure if we should mention the Onchain Foundation as part of governance, as it is not clear was exact responsibilities the Foundation has

Reviewer: > The Lisk project is governed mainly by the [Lisk security council](#lisk-security-council) and the [Lisk DAO](#lisk-dao), which allows Lisk token holders to participate together in the governance of the Lisk project.

but from this sentence, it sounds like `Lisk Security Concncil` is responsible for the governance of the `Lisk Project` which is not true. 
Also, Lisk DAO can propose something to overall governance, but it's not main party of governance i would say.
For example, Lisk DAO cannot decide about the upgrade, protocol related stuff, only suggestion

Author: Ok, I updated the phrase to include Onchain foundation, please check.

Reviewer: it looks good to me, but for this i think it's better to ask opinion from wider range. Let me quickly check others

Reviewer: From @janhack, 
> I don't think the security council plays a role in the the decision-making for the Lisk Project and I would therefore not mention it when talking about governance. Unlike the OP Security Council and security councils for other L2s, we don't have any upgrade keys for our L2 and the security council therefore does not have any role in protocol updates. Rather, it only has upgrade keys to Claiming, Staking and Vesting Wallet. We don't have any intention to upgrade those and the only time the security council would intervene is in case of incident. Here it also only acts on behalf of the Onchain Foundation.
Moreover, I think we should be careful how we formulate things. The formulation above makes it sound like the Onchain Foundation has all power and the Lisk DAO is only "supporting", which sounds like it has no power at all. This is not the case (The Lisk DAO has full control over the DAO treasury, for instance) and I would therefore reformulate the statement.

```suggestion
The Lisk project is governed by the Lisk DAO and Onchain Foundation. The Onchain Foundation defines the vision and strategy for the project together with the Lisk team, funds its development and is a stakeholder in the Lisk DAO via the Lisk Team delegate. The Lisk DAO allows any LSK token holder that stakes their tokens to participate in onchain governance for the Lisk project. The Lisk DAO decides how to spend funds in its significant treasury and can pass general proposals about protocol parameters or the project direction.
```

Also, https://docs.lisk.com/governance/overview we already have good information already

Author: @shuse2 I accepted your suggestion 👍

:::info
Every LSK token holder who participates in the DAO is rewarded with [staking rewards](../user/staking/overview#staking-rewards), to incentivize participation in the Lisk governance.
:::

## Security council

The security council is a 3-of-5 multi-signature wallet with key members of Lisk team and Onchain Foundation.

The purpose of the security council is to be responsible for smart contract upgrades in case a security vulnerability is discovered.
Therefore, it is crucial that in case of a security incident the security council can act promptly, i.e., at least 3 members are available for signing a contract upgrade.

The security council has the power to upgrade the following contracts:

- Claims contract
- Governance contracts
- Staking and Rewards contracts
- all Vesting contracts

> - Airdrop contract

Reviewer: Airdrop contract is not upgradable

Author: updated line 36 to 
>The security council controls the following contracts:

Reviewer: im not sure if `control` is a right word, but i also don't have other idea...

The ERC-20 token contract cannot be upgraded by the security council, but the security council has the special power to add accounts that can burn LSK tokens.
This is only planned to be used in case the community decides to burn the 100M LSK allocated to the Lisk DAO.

## Lisk Admin Addresses

> Details of **Admin account** addresses on 🟢 **Ethereum Mainnet** and 🟡 **Sepolia Testnet** are as follows.

Reviewer: Good to mention those Gelato controlled account will be transfered to the OP security council in the near future

> <table >

Reviewer: ```suggestion
<table>
```

import TabItem from '@theme/TabItem';

> # Governance overview

Reviewer 2: The following link is not working!
![image](https://github.com/user-attachments/assets/c81a6800-2a99-4a35-856a-a7ea01fd3847)

The Lisk project is governed mainly by the [Lisk security council](#security-council) and the [Lisk DAO](#lisk-dao), which allows Lisk token holders to participate together in the governance of the Lisk project.

:::info
Every LSK token holder who participates in the DAO is rewarded with [staking rewards](../user/staking/overview#staking-rewards), to incentivize participation in the Lisk governance.
:::

## Lisk Security Council

The Lisk security council is a 3-of-5 multi-signature wallet with key members of Lisk team and Onchain Foundation.

The purpose of the security council is to be responsible for smart contract upgrades in case a security vulnerability is discovered.

The security council controls the following contracts:

- Claims contract
- Governance contracts
- Staking and Rewards contracts

> - all Vesting contracts

Reviewer 2: ```suggestion
- All Vesting contracts
```

- Airdrop contract

The ERC-20 token contract cannot be upgraded by the security council, but the security council has the special power to add accounts that can burn LSK tokens.
This is only planned to be used in case the community decides to burn the 100M LSK allocated to the Lisk DAO.

## Lisk Admin Addresses

Details of **Admin account** addresses on 🟢 **Ethereum Mainnet** and 🟡 **Sepolia Testnet** are as follows.

:::note

> The Gelato controlled accounts will be transfered to the [OP security council](https://gov.optimism.io/t/intro-to-optimisms-security-council/6885) in the near future.

Reviewer 2: ```suggestion
The Gelato-controlled accounts will be transferred to the [OP security council](https://gov.optimism.io/t/intro-to-optimisms-security-council/6885) in the near future.
```

:::

> <table>

Reviewer: table is still not working?

Author: works for me: https://liskhq.github.io/lisk-documentation/governance/overview

Reviewer: ah now it's working 👍

  <thead>
    <tr>
      <th scope="col">Admin Role</th>
      <th scope="col">Controlled/Owned by</th>
      <th scope="col" colSpan="2">Address</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row" rowSpan="2">L1 Proxy Admin</th>
      <th scope="row" rowSpan="2">Gelato</th>
      <td>[0xeC432c4F1d0E12737f3a42a459B84848Af979b2d](https://etherscan.io/address/0xeC432c4F1d0E12737f3a42a459B84848Af979b2d)</td>
      <td>🟢</td>
    </tr>
    <tr>
    <td>[0x5Db9F05921d8d5a6a157F6f49c411cc0e46c6330](https://sepolia.etherscan.io/address/0x5Db9F05921d8d5a6a157F6f49c411cc0e46c6330)</td>
    <td>🟡</td>
    </tr>

> 

Reviewer 2: Removing whitespaces between the `<tr>` tags will also rectify rendering issues on Markdown.
```suggestion
```

    <tr>
      <th scope="row" rowSpan="2">SystemConfig</th>
      <th scope="row" rowSpan="2">Gelato</th>
      <td>[0x05f23282FFDCA8286E4738C1aF79079f3d843750](https://etherscan.io/address/0x05f23282FFDCA8286E4738C1aF79079f3d843750)</td>
      <td>🟢</td>
    </tr>
    <tr>
    <td>[0xF54791059df4a12BA461b881B4080Ae81a1d0AC0](https://sepolia.etherscan.io/address/0xF54791059df4a12BA461b881B4080Ae81a1d0AC0)</td>
    <td>🟡</td>
    </tr>

> 

Reviewer 2: ```suggestion
```

    <tr>
      <th scope="row" rowSpan="2">Batcher</th>
      <th scope="row" rowSpan="2">Gelato</th>
      <td>[0xa6Ea2f3299b63c53143c993d2d5E60A69Cd6Fe24](https://etherscan.io/address/0xa6Ea2f3299b63c53143c993d2d5E60A69Cd6Fe24)</td>
      <td>🟢</td>
    </tr>
    <tr>
    <td>[0x246E119a5BcC2875161b23E4e602e25cEcE96E37](https://sepolia.etherscan.io/address/0x246E119a5BcC2875161b23E4e602e25cEcE96E37)</td>
    <td>🟡</td>
    </tr>

> 

Reviewer 2: ```suggestion
```
